docs/os_list.json.template

```diff
@@ -0,0 +1,61 @@
+{
+    "imager": {
+        "latest_version": "1.8.5",
+        "url": "https://www.raspberrypi.com/software/",
+        "devices": [
+            {
+                "name": "No filtering",
+                "tags": [],
+                "default": false,
+                "description": "Show every possible image",
+                "matching_type": "inclusive"
+            },
+            {
+                "name": "Raspberry Pi 5",
+                "tags": [
+                    "pi5-64bit"
+                ],
+                "icon": "https://downloads.raspberrypi.com/imager/icons/RPi_5.png",
+                "description": "Raspberry Pi 5, 500, and Compute Module 5",
+                "matching_type": "exclusive"
+            }
+        ]
+    },
+    "os_list": [
+        {
+            "name": "AOS",
+            "description": "AOS for Raspberry Pi",
+            "icon": "https://avatars.githubusercontent.com/u/96244376?s=200&v=4",
+            "subitems": [
+                {
+                    "name": "AOS",
+                    "description": "Preconfigured AOS image for SD card boot",
```

Comment:
SD is used in both cases. The difference is to use either USB flash or NVME. So, name should be:
"Preconfigured AOS image with rootfs on USB drive" and "Preconfigured AOS image with rootfs on NVME drive"